kevm-pyk/src/tests/unit/test_foundry_list.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import json
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from pyk.proof.reachability import APRProof
+
+from kevm_pyk.solc_to_k import Contract
+from tests.unit.utils import TEST_DATA_DIR
+
+if TYPE_CHECKING:
+    from typing import Final
+
+LIST_DATA_DIR: Final = TEST_DATA_DIR / 'foundry-list'
+LIST_OUT: Final = LIST_DATA_DIR / 'out/'
+LIST_APR_PROOF: Final = LIST_OUT / 'apr_proofs/'
+LIST_EXPECTED: Final = LIST_DATA_DIR / 'foundry-list.expected'
+
+
+def contracts() -> dict[str, Contract]:
```

Comment:
This logic should not be replicated here. Instead, consider mocking `Foundry`.

Reply:
Much better, thanks !